sources/editor/Stride.Core.Assets.Editor/ViewModel/AssetCollectionViewModel.cs

```diff
@@ -537,31 +535,99 @@ private string ComputeNamespace(DirectoryBaseViewModel directory)
             }
         }
 
-        private async Task<List<AssetViewModel>> InvokeAddAssetTemplate(LoggerResult logger, string name, DirectoryBaseViewModel directory, TemplateAssetDescription templateDescription, IEnumerable<UFile> files, PropertyContainer? customParameters)
+        private async Task<string> GetAssetCopyDirectory(DirectoryBaseViewModel directory, UFile file)
         {
-            List<AssetViewModel> newAssets = new List<AssetViewModel>();
-            if (files is not null)
-            {
-                foreach (var file in files)
+            string path = directory.Path;
+            var message = Tr._p("Message", "Do you want to place the resource in the default location ?").ToFormat(file.FullPath);
+            string finalPath = directory.Package.Package.ResourceFolders[0] + "/" + path + file.GetFileName();
+            var pathResult = await Dialogs.MessageBox(message, MessageBoxButton.YesNo, MessageBoxImage.Question);
+            if (pathResult == MessageBoxResult.No)
+            {
+                bool done = false;
+                do
                 {
-                    bool inResourceFolder = false;
-                    foreach (var resourceFolder in directory.Package.Package.ResourceFolders)
+                    var fileDialog = Dialogs.CreateFileSaveModalDialog();
+                    fileDialog.Filters = new List<FileDialogFilter>() { new FileDialogFilter("", file.GetFileExtension()) };
+                    fileDialog.InitialDirectory = directory.Package.Package.ResourceFolders.FirstOrDefault().FullPath.Replace("/", "\\");
```

Comment:
While the editor doesn't run on anything else but windows for now, might as well reduce issues we might have when migrating.
https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/8.0/file-path-backslash
> Backslash ( \ ) characters are valid in directory and file names on Unix

You can use `Path.GetFullPath(myPath)` to convert into the OS's directory separator.